src/timer.c

```diff
@@ -34,16 +34,18 @@
 
 #if (defined WIN32 || _WIN64)
 
+#include <Windows.h>

```

Comment:
Could you make this a lower-case `w`, for the sake of cross-compiling from a case-sensitive OS?